apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -29,7 +32,7 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
         size={'xs'}
       />
     </div>
-  );
+  ) : null;
```

Comment:
Doesn't need to be changed, but I think this'd read more clearly with a negated early return instead:

```
if (isReadOnly) {
  return null
}
...
return <div><Button>...etc
```

Reply:
updated!